glib/src/auto/functions.rs

```diff
@@ -54,6 +54,13 @@ pub fn base64_encode(data: &[u8]) -> crate::GString {
 //    unsafe { TODO: call ffi:g_base64_encode_step() }
 //}
 
+#[doc(alias = "g_blow_chunks")]
+pub fn blow_chunks() {
```

Comment:
This one is deprecated and we should just not generate at all

Reply:
Probably worth an upstream issue. That one wasn't even in the docs anymore :)

Reply:
Removed it, see last commit